subsys/bluetooth/host/hci_core.c

```diff
@@ -802,6 +834,13 @@ static void le_conn_complete(struct net_buf *buf)
 		}
 	}
 
+	if (BT_FEAT_LE_DLE(bt_dev.le.features)) {
+		err = hci_le_set_data_len(conn);
+		if (!err) {
+			goto done;
```

Comment:
Just checking: is this "if (!err) goto done;" logic correct for the DLE and set_phy() branches? The intention of it for the preceding read_remote_features() branch is that we continue the remaining steps once we get the remote features complete event.